ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -489,6 +495,20 @@
         }
     });
 
+    function registerRestEndpointUrlValidator() {
+        $(window).adaptTo("foundation-registry").register("foundation.validation.validator", {
+            selector: "[data-rest-endpoint-url-validation]",
+            validate: function(el) {
+                var url = el.value;
+                var absoluteUrlPattern = /^(?:[a-z]+:)?\/\//i;  // Pattern to match absolute URLs
```

Comment:
This regex would also match the following string `//example.com (protocol-relative URL)`, I don't think we support this

Reply:
Sure updating the regex to below, the URL starts with http:// or https://.
It has a valid domain (e.g., example.com or sub.example.com).
It optionally includes a port number (e.g., :8080).
It optionally includes a path (e.g., /path/to/resource).